sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -38,3 +38,43 @@ If no `options` parameter is supplied, the report is exported using the default
 
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
+
+### getSelectedData(options?: {formatData: boolean | "datesOnly"}): ReportObjectResultData[]
+
+Returns a user's selection data from the visual. Returns an empty array if the visual has no selections.
+
+#### Arguments
+
+`options` is an optional options bundle for customizing the returned data. The following options are supported:
+
+- `formatData` for specifying the format of the returned data. It can be passed the following values:
```

Comment:
'formData' specifies the format of the returned data. It can be passed using the following values:

Reply:
With the second sentence I was intending to say something like "the `formatData` variable can be one of the following values", so "passed using the following values" doesn't sound accurate to me.

I removed that part of the sentence entirely since it seems like the values following the variable is explanation enough, but let me know if you have a different idea